jq_dict_parser.go

```diff
@@ -27,8 +29,8 @@ func NewJQDictParser(dict map[string]string) map[string]JQFieldParser {
 	return output
 }
 
-func (p JQDictParser) Run(data string) (map[string]string, error) {
-	output := make(map[string]string)
+func (p JQDictParser) Run(data string) ([]opslevel.PropertyInput, error) {
+	output := make([]opslevel.PropertyInput, 0)
```

Comment:
Deduplication not necessary here since properties are defined using a JSON object (no 2 definitions can be the same on one service.)